internal/monitor/features/integration.feature

```diff
@@ -621,6 +707,29 @@ Feature: Integration Test
       | ""         | "3-5"       | "1-1" | "0-0" | "isilon" | "isilon"   | "one-third" | "zero"  | "reboot" | 240      | 900        | 900     | 900           |
     # | ""         | "3-5"       | "2-2" | "0-0" | "isilon" | "isilon"   | "one-third" | "zero"  | "reboot" | 240      | 240        | 300     | 600           |
     # | ""         | "5-10"       | "1-1" | "0-0" | "isilon" | "isilon"   | "one-third" | "zero"  | "reboot" | 1200      | 2000       | 2000     | 2000           |
+  
+  @powerstore-integration
+  Scenario Outline: Basic node failover testing using test StatefulSet pods (node slow reboots)
+    Given a kubernetes <kubeConfig>
+    And cluster is clean of test pods
+    And wait <nodeCleanSecs> to see there are no taints
+    And <podsPerNode> pods per node with <nVol> volumes and <nDev> devices using <driverType> and <storageClass> in <deploySecs>
+    Then validate that all pods are running within <deploySecs> seconds
+    When I fail <workers> worker nodes and <primary> primary nodes with <failure> failure for <failSecs> seconds
+    Then validate that all pods are running within <runSecs> seconds
+    And labeled pods are on a different node
+    And the taints for the failed nodes are removed within <nodeCleanSecs> seconds
+    Then finally cleanup everything
+    Examples:
+      | kubeConfig | podsPerNode | nVol  | nDev  | driverType | storageClass | workers     | primary | failure  | failSecs | deploySecs | runSecs | nodeCleanSecs |
+       #Small number of pods, increasing number of vols and devs
+      | ""         | "1-2"       | "1-1" | "0-0" | "pstore" | "powerstore-nfs"   | "one-third" | "zero"  | "reboot" | 240      | 600        | 600     | 600          |
+   #   | ""         | "1-2"       | "2-2" | "0-0" | "pstore" | "powerstore-nfs"   | "one-third" | "zero"  | "reboot" | 600      | 900        | 900     | 900           |
```

Comment:
Delete if not required, don't have keep commented test

Reply:
Added iscsi & nvme types.
Removed commented lines.